date/clone.test.ts

```diff
@@ -0,0 +1,11 @@
+/* eslint-env jest */
+// @ts-ignore ambiguous import
+import clone from "./clone.ts";
+
+describe("clone", () => {
+  it("should return cloned date", () => {
+    const date = new Date("2019-04-24T13:54:33.232Z");
+
+    expect(clone(date)).not.toBe(date);
```

Comment:
Please assert value equality as well.

```suggestion
    const cloned = clone(date);
    expect(cloned).not.toBe(date);
    expect(cloned).toEqual(date);
```